src/components/footer.js

```diff
@@ -0,0 +1,257 @@
+    height: 36px;
+    margin-top: 10px;
+    border: solid #626262 1px;
+  }
+
+  @media (max-width: ${dimensions.maxwidthTablet}px) {
+    input {
+      width: 80%;
+      height: 36px;
+      margin-top: 10px;
+      border: solid #626262 1px;
+    }
+  }
+
+  input::placeholder {
+    padding-left: 14px;
+  }
+
+  @media (min-width: ${dimensions.maxwidthTablet}px) {
+    padding-top: 50px;
+    position: absolute;
+    right: 62px;
+  }
+`
+
+const BottomLinks = styled.div`
+  a {
+    text-decoration: none;
+    color: black;
+  }
+  ul {
+    display: flex;
+    justify-content: center;
+    list-style: none;
+    padding-bottom: 30px;
+  }
+
+  li {
+    font-size: 12px;
+    padding-right: 28px;
+  }
+
+  @media (max-width: ${dimensions.maxwidthTablet}px) {
+    ul {
+      display: flex;
+      justify-content: center;
+      list-style: none;
+      padding-top: 80px;
+      padding-bottom: 12px;
+    }
+  }
+`
+
+const AddressSection = styled.div`
+  width: 177px;
+  padding-top: 16px;
+`
+const ScoutSection = styled.div`
+  margin-top: 32px;
+  color: #ffffff;
+
+  @media (max-width: ${dimensions.maxwidthTablet}px) {
+    padding-bottom: 48px;
+  }
+`
+
+const EmailFormHeader = styled.div``
+
+export default function footer({ data }) {
+  return (
+    <StaticQuery
+      query={graphql`
+        query prismicFooter {
+          prismicFooter {
+            data {
+              email
+              social_media {
+                social_icon {
+                  url
+                }
+                social_link
+              }
+            }
+          }
+        }
+      `}
+      render={data => (
+        <FooterContainer>
+          <FooterMainContent>
+            <SocialSection>
+              <FooterHeader>Powerhouse</FooterHeader>
+              <AddressSection>
+                123 Lorem Ipsum Dolor Tempor Incididunt YZ, 12345 123-456-7890
+              </AddressSection>
+              {/* <EmailText>{data.prismicFooter.data.email}</EmailText> */}
+            </SocialSection>
+
+            <LinkSection>
+              <LinkColumn>
+                <ul>
+                  <li>
+                    <a href="/about">Home</a>
+                  </li>
+                  <li>
+                    <a href="/about">About</a>
+                  </li>
+                  <li>
+                    <a href="/about">Process</a>
+                  </li>
+                </ul>
+              </LinkColumn>
+
+              <LinkColumn>
+                <ul>
+                  <li>
+                    <a href="/about">Products</a>
+                  </li>
+                  <li>
+                    <a href="/about">Solutions</a>
+                  </li>
+                  <li>
+                    <a href="/about">FAQ</a>
+                  </li>
+                </ul>
+              </LinkColumn>
+            </LinkSection>
+
+            <EmailSection>
+              <form>
+                <input
+                  type="text"
```

Comment:
Could you please add aria labels or names to these labels + forms? [here is a nifty document](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute)